File: Prime/prime.c
#include <stdio.h>

int main()
{
	int a;
	printf("Enter a number : ");
	scanf("%d", &a);

	for (int i=2; i<a/2; i++)
	{
		if(a%i == 0)
		{
			printf("Number is not prime. \n");
			return 0;
		}
	}
	printf("Number is prime number \n");
}

File: 2nd Year/OOPS_LAB/src/Program3/prime.java
package Program3;  // Do not write this while copying in Lab Record/ executing on your own
import java.util.Scanner;

public class prime {

	public static void main(String[] args) 
	{
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter a number : ");
		int n = sc.nextInt();
		
		for(int i=2; i<n;i++)
		{
			if(n%i == 0)
			{
				System.out.println("The number is not a prime number");
				System.exit(0);
			}
		}
		System.out.println("The number is  a prime number");
		sc.close();
	}

}
